engine/apps/alerts/tasks/check_escalation_finished.py

```diff
@@ -106,10 +108,17 @@ def check_alert_group_personal_notifications_task(alert_group_id) -> None:
         notification_step=UserNotificationPolicy.Step.NOTIFY,
     ).count()
 
+    # sent SMS messages are considered completed for our purpose here
+    # (ie. do not wait for Twilio delivered confirmation)
+    sent_but_not_delivered_sms = SMSRecord.objects.filter(
+        represents_alert_group_id=alert_group_id,
+        twilioapp_twiliosmss__status=TwilioSMSstatuses.SENT,
+    ).count()
+
     base_msg = f"Alert group {alert_group_id}"
+    completed += sent_but_not_delivered_sms
     delta = triggered - completed
     if delta > 0:
-        # TODO: when success notifications are setup for every backend, raise exception here
```

Comment:
do we want to raise an exception here still?

Reply:
Good question :+1: . Not now, we changed the approach. The original idea was for this to be part of the alert group auditor checks, so raising an exception would make the escalation checker heartbeat fail, but we switched this to be a separated thing to avoid overloading the alert group auditor checks and mix escalation with personal notifications (personal notifications check run on separated tasks).